content/200-concepts/100-components/250-preview-features/050-client-preview-features.mdx

```diff
@@ -12,6 +12,7 @@ The following [Preview](/about/prisma/releases#preview) feature flags are availa
 - [`referentialIntegrity`](/concepts/components/prisma-schema/relations/referential-integrity) since release [3.1.1](https://github.com/prisma/prisma/releases/tag/3.1.1) - [Submit feedback](https://github.com/prisma/prisma/issues/9380)
 - [`fullTextIndex`](/concepts/components/prisma-schema/indexes#full-text-indexes-mysql-and-mongodb) since release [3.6.0](https://github.com/prisma/prisma/releases/tag/3.6.0) - [Submit feedback](https://github.com/prisma/prisma/issues/10539)
 - [`metrics`](/concepts/components/prisma-client/metrics) since release [3.15.0](https://github.com/prisma/prisma/releases/tag/3.15.0) - [Submit feedback](https://github.com/prisma/prisma/issues/13579)
+- [`tracing`](/concepts/components/prisma-client/opentelemetry-tracing) since release [4.2.0](https://github.com/prisma/prisma/releases/tag/4.2.0) - [Submit feedback](https://github.com/prisma/prisma/issues/14640)
```

Comment:
Do we have an open issue for the preview feature feedback as well?

Reply:
It links to https://github.com/prisma/prisma/issues/14640

Reply:
:facepalm: thanks